recipes/asl/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "asl" %}
```

Comment:
it was asl (ampl solver library) when it was on netlib, and it's still there under that name i think. the github repo was named ampl mp, with the existing asl as one component, when @vitaut started working on the ampl codebase and adding a lot of new C++ functionality, a cmake build system, etc. so the github repo has more than just the libasl in it, but established library consumers like ipopt don't necessarily need the new stuff. some more recent ampl solver interfaces do, like gecode and a few others I think